python/paddle/v2/framework/layers.py

```diff
@@ -202,6 +190,30 @@ def concat(input, axis, program=None, init_program=None):
     return out
 
 
+def sums(input, program=None, init_program=None):
+    helper = LayerHelper('sum', **locals())
+    if not isinstance(input, list) and not isinstance(input, tuple):
+        input = [input]
```

Comment:
There is no need to do the converting here. Operator's constructor will do this.